lib/grammers-client/src/types/update.rs

```diff
@@ -15,12 +15,19 @@ use std::sync::Arc;
 pub enum Update {
     /// Occurs whenever a new text message or a message with media is produced.
     NewMessage(Message),
+    /// Occurs when a message is updated.
+    MessageEdited(Message),
+    /// Occurs when a message is deleted.
+    MessageDeleted(MessageDeletion),
     /// Occurs when Telegram calls back into your bot because an inline callback button was
     /// pressed.
     CallbackQuery(CallbackQuery),
     /// Occurs whenever you sign in as a bot and a user sends an inline query such as
     /// `@bot query`.
     InlineQuery(InlineQuery),
+    /// Raw events are not actual events.
+    /// Instead, they are the raw Update object that Telegram sends. You normally shouldn’t need these.
```

Comment:
Could we add a note like "the library can move events out of this variant into a new one across minor versions"? I don't mind the exact wording, but I want it to be clear that this variant pretty much means "WIP use this as a workaround".